src/cursor.c

```diff
@@ -142,26 +152,12 @@ int Cursors_CollectIdle(CursorList *cl) {
   return rc;
 }
 
-void CursorList_AddSpec(CursorList *cl, const char *k, size_t capacity) {
-  CursorSpecInfo *info = findInfo(cl, k);
-  if (!info) {
-    info = rm_malloc(sizeof(*info));
-    info->keyName = rm_strdup(k);
-    info->used = 0;
-    dictAdd(cl->specsDict, (void *)k, info);
-  }
-  info->cap = capacity;
-}
-
-void CursorList_RemoveSpec(CursorList *cl, const char *k) {
-  CursorSpecInfo *info = findInfo(cl, k);
-  if (info) {
-    dictDelete(cl->specsDict, k);
-    rm_free(info->keyName);
-    rm_free(info);
-  }
+void Cursors_initSpec(IndexSpec *spec, size_t capacity) {
```

Comment:
I think this should be a part of the spec creation and not related to the cursor h/c files